frontend/src/theme/components/Tooltip.ts

```diff
@@ -0,0 +1,26 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+export const TOOLTIP_THEME_KEY = 'CustomTooltip'
+export const CustomTooltip: ComponentMultiStyleConfig = {
+  parts: ['tooltip', 'wrapper'],
+  baseStyle: {
+    tooltip: {
+      '--tooltip-bg': 'var(--chakra-colors-secondary-700)',
```

Comment:
I see that chakra's own tooltip has a `bg` prop in addition to this. Is there a reason `bg` is not included here?

Reply:
I think `bg` uses the `--tooltip-bg` value so I think overriding `--tooltip-bg` is sufficient?